django_project/sawps/admin.py

```diff
@@ -28,6 +32,11 @@ class ExtendedGroupInline(admin.StackedInline):
 
 
 class GroupAdmin(BaseGroupAdmin):
+
+    def save_model(self, request, obj, form, change):
+        with disconnected_signal(post_save, save_extended_group, Group):
```

Comment:
I disconnect the signal here because it creates multiple groups. However, we still want to retain the signals.